var/spack/repos/builtin/packages/py-json-get/package.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyJsonGet(PythonPackage):
```

Comment:
I think this package was already merged and it accidentally got sucked into this PR. Can you remove it from this PR?